lib/pliny/librato/metrics/backend.rb

```diff
@@ -21,10 +22,14 @@ def report_measures(measures)
 
       def _report_counts(counts)
         ::Librato::Metrics.submit(expand(:counter, counts))
+      rescue => error
```

Comment:
`rescue => error` and `rescue StandardError => error` are the same thing. I think you're thinking of the rule to never do `rescue Exception => error` which would catch interrupt signals but [that's not this](https://robots.thoughtbot.com/rescue-standarderror-not-exception).